docs/pages/contributing/documentation/reference.mdx

```diff
@@ -275,7 +285,9 @@ require more than lines, coloring, and text must use another solution.
 
 ## Notices
 
-<Notice type="tip">Notice content.</Notice>
+<Notice type="tip">
+Notice content.
+</Notice>
 
```

Comment:
This is my opinion 😄 
Think twice before adding any type of special notice. A paragraph highlighted as a note, tip, caution, or warning can be distracting and too many of them make text unreadable. 

Because they are generally disruptive to the reader’s experience, ask yourself if it is really necessary to stop the reader’s forward progress by adding a Note, Caution, or Tip component. If you use them at all, use them sparingly.